setup.py

```diff
@@ -62,7 +62,7 @@ def generate_proto_sources(self):
 
 setup(
     name = "CuraEngineGRPC",
-    version = "0.1.1",
+    version = "0.1.2",
```

Comment:
Right, so `0.1.1` would already have been correct then, since this introduces a change.

(Unless we want to also keep increasing the minor instead of patch, but since we're still in v0 that won't be necessary.)

Will change back.

Reply:
We also haven't released yet